.github/workflows/linked-issue-check.yml

```diff
@@ -0,0 +1,130 @@
+# for more information.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Linked Issue Check
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  linked-issue-check:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: read
+      pull-requests: read
+    outputs:
+      linked-issues: ${{ steps.get-linked-issues.outputs.issues }}
+      linked-issues-count: ${{ steps.get-linked-issues.outputs.linked_issues_count }}
+    steps:
+      - name: Get Linked Issues
+        id: get-linked-issues
+        uses: nearform-actions/github-action-check-linked-issues@v1
+        with:
+          custom-body-comment: 'This PR is not linked to any issues. Please consider adding the corresponding issues if they exist, in the pull request description.'
+        continue-on-error: true
+  closing-issues:
+    name: ${{ matrix.linked-issue }}
+    strategy:
+      fail-fast: false
+      matrix:
+        linked-issue: ${{ fromJSON(needs.linked-issue-check.outputs.linked-issues) }}
+    runs-on: ubuntu-latest
+    needs: linked-issue-check
+    defaults:
+      run:
+        working-directory: ${{ github.workspace }}/app
+    if: ${{ fromJSON(needs.linked-issue-check.outputs.linked-issues-count) > 0 }}
+    steps:
+      #----------------------------------------------------------------------
+      # Install all tools and check configuration
+      #----------------------------------------------------------------------
+      - name: 'SETUP: Checkout generator-jhipster'
+        uses: actions/checkout@v4
+        with:
+          path: generator-jhipster
+          fetch-depth: 1
+      - uses: ./generator-jhipster/.github/actions/setup-default-node-java
+      - uses: jhipster/actions/setup-keycloak-hostname@v0
+      - uses: jhipster/actions/setup-git@v0
+      - uses: jhipster/actions/restore-cache@v0
+        with:
+          npm: true
+          maven: true
+          gradle: true
+      #----------------------------------------------------------------------
+      # Install JHipster and generate project+entities
+      #----------------------------------------------------------------------
+      - name: 'GENERATION: install JHipster'
+        run: $JHI_SCRIPTS/10-install-jhipster.sh
+      - name: 'GENERATION: project'
+        id: project
+        run: jhipster from-issue "$(echo ${{ matrix.linked-issue }} | cut -d "#" -f 2)" --no-code-workspace
```

Comment:
```suggestion
        run: jhipster from-issue "${{ matrix.linked-issue }}" --no-code-workspace
```

Split the owner/repo/issue here:
https://github.com/jhipster/generator-jhipster/blob/94b9f70751fd4731232e41c42a75b555345c4ca0/.blueprint/from-issue/generator.mjs#L55-L57

Reply:
The format is owner/repo#number.
![image](https://github.com/jhipster/generator-jhipster/assets/1417183/68d0014b-47ac-44b1-81cc-4cb94d5bd626)

Reply:
So what I am thinking is that, if we set, `jhipster from-issue "${{ matrix.linked-issue }}" --no-code-workspace` then the code would fail as the `from-issue` arg should be just the issue number. Am I missing something? 🤔

Reply:
You are extracting the issue number with `cut` command.
So removing the cut command like the suggested change it will pass the entire repo spec.
`jhipster from-issue "${{ matrix.linked-issue }}" --no-code-workspace`

Reply:
Then we need to split inside js code:
```
if (this.issue.includes('#')){
  let split = this.issue.split('/');
  this.owner = split[0];
  split = split[1].split('#');
  this.repo = split[0];
  this.issue = split[1];
}

Reply:
Ah, sorry now I understand what you meant there. Just tweaked the code to incorporate this change. 👍🏽